dask_ml/model_selection/methods.py

```diff
@@ -20,7 +20,7 @@
 
 # Copied from scikit-learn/sklearn/utils/fixes.py, can be removed once we drop
 # support for scikit-learn < 0.18.1 or numpy < 1.12.0.
-if LooseVersion(np.__version__) < "1.12.0":
+if packaging.version.parse(np.__version__) < packaging.version.parse("1.12.0"):
```

Comment:
These individual checks can probably be moved to the `_compat` module similar to how it's done in other parts of the codebase.

Reply:
Alternatively, given that today the [minimum supported NumPy version for Dask is 1.18](https://github.com/dask/dask/blob/f34f1afd407d4371901a7ccb14a4a6b86c4cb4c2/setup.py#L13), we could probably just bump the minimum supported version for Dask-ML too (though let's handle that in a separate PR)